opensearch-project/OpenSearch-Dashboards
# [navigation-next] feat: add recent works in new homepage

### Description


Add recent works section in new home page.

### Issues Resolved




## Screenshot


### Workspace enabled
https://github.com/user-attachments/assets/b0fa655f-4328-41fd-bc47-1818c18e7a27

### Workspace disabled
https://github.com/user-attachments/assets/b7b9f7dd-a913-4a1c-b161-df69e0ad8151

### Empty state
<img width="1727" alt="image" src="https://github.com/user-attachments/assets/426a9dff-0f74-4f1d-bd5a-35df716a9223">

## Testing the changes


- Go to the advanced settings page to enable new home page.
- Go to home page, you should see the recent items section is displayed in home page.

## Changelog

- feat: [navigation-next] add recent works in new homepage

### Check List
